src/remediations/fifi.js

```diff
@@ -37,6 +39,12 @@ exports.checkSmartManagement = async function (remediation, smart_management) {
         return false;
     }
 
+    const rhcStates = await configManager.getCurrentState();
```

Comment:
As this is unrelated to smart management it would be a bit cleaner either to do the check outside of the function or rename the function to have a more general name. Also, since this is a API call there is a potential to optimize performance by making the call in parallel with other slow operations (API calls, db queries, ...).